### ISSUE TITLE
Catalog Error: Could not rename

### ISSUE THREAD

**Issue by johalnes** [jwills/dbt-duckdb#103]
Hi,

really impressed with how fast you implemented attach database! Looking forward to start working more with dbt + duckdb 😄 

I'm trying it out now by mimicking databricks medallion with a separate database for bronze, silver and gold. First run always works like a charm, but with subsequent runs I get  "Catalog Error: Could not rename". Could it be that there is some kind of restrictions with renaming across attached databases? 

Changing back to only use the "main" database, the one added without attach, it works every time.

I'll try to force the same error from jaffle shop example or something similar, but probably won't have time until the weekend.
![error](https://user-images.githubusercontent.com/8577145/219069548-56f134ad-a8cd-40a5-94b4-23bf8aab419f.png)

**Comment by jwills** [comment]
Hey @johalnes, thanks so much for trying this out and reporting this issue! I have a couple of thoughts on what could be going on here and a little integration test setup I can use to help me figure it out!

**Comment by johalnes** [comment]
Perfect!

Give me a heads up if you want me to install from a development branch and see if it’s fixed.

**Comment by johalnes** [comment]
Hi,

I've tried to force this error using the jaffle example now, but with the same results as you. Works every time! 

The project giving this error is actually a really large one which is in production in snowflake, and where disabled a lot to get single error. So it's likely that I've messed up a config somewhere... 
I'll start from scratch and if it occurs again and I'm able to isolate the issue I'll reopen! Thanks for helping out, and keep up the good work with this project 😄
